include/boost/url/scheme.hpp

```diff
@@ -12,6 +12,7 @@
 
 #include <boost/url/detail/config.hpp>
 #include <boost/url/string_view.hpp>
+#include <cinttypes>
```

Comment:
huh... why didn't you just make it unsigned short?

Reply:
To make it coherent with `std::uint16_t url_base_view::port_number() const noexcept`.